packages/web-runtime/src/pages/accessDenied.vue

```diff
@@ -39,11 +47,20 @@ export default defineComponent({
     const logoImg = computed(() => {
       return store.getters.configuration.currentTheme.logo.login
     })
+
+    const loggedOutHelpUrl = computed(() => {
+      return (
+        store.getters.configuration.commonTheme.loggedOutHelpUrl ||
+        store.getters.configuration.options.loggedOutHelpUrl
+      )
+    })
     const cardTitle = computed(() => {
-      return $gettext('Logged out')
+      return $gettext('Not logged in')
     })
     const cardHint = computed(() => {
-      return $gettext('You were automatically logged out for security reasons.')
+      return $gettext(
+        'This could be because of a routine safety log out, or because your account is either inactive or not yet authorized for use. Please try logging in after a while or seek help from your Administraton.'
```

Comment:
```suggestion
        'This could be because of a routine safety log out, or because your account is either inactive or not yet authorized for use. Please try logging in after a while or seek help from your Administrator.'
```

Reply:
I'd like to spawn @phil-davis here for a native speaker opinion 🤓

Reply:
oops